src/ontology/mondo-ingest.Makefile

```diff
@@ -107,6 +107,7 @@ $(COMPONENTSDIR)/doid.owl: $(TMPDIR)/doid_relevant_signature.txt | component-dow
 		remove -T $(TMPDIR)/doid_relevant_signature.txt --select complement --select "classes individuals" --trim false \
 		query \
 			--update ../sparql/fix_omimps.ru \
+			--update ../sparql/fix_mim.ru \
```

Comment:
## Approve
I wasn't sure about the last version of these changes from the other day, but what you have in this PR immediately strikes me as clear and correct! And the [build](https://github.com/monarch-initiative/mondo-ingest/pull/569) results also seem to corroborate.